src/main.ts

```diff
@@ -1,25 +1,29 @@
 // Generator Modules
-import {picTextGenerator} from './pages/pic-text';
 import {
-  addGradientTextListener,
-  gradientTextGenerator,
-} from './pages/gradient-text';
+  addAnimationListener,
+  animationGenerator,
+  displayAnimationPreview,
+} from './pages/animation';
 import {
-  gradientBorderGenerator,
-  addGradientBorderListener,
-} from './pages/gradient-border';
+  addBorderRadiusListener,
+  borderRadiusGenerator,
+} from './pages/border-radius';
+import {addBoxShadowListener, boxShadowGenerator} from './pages/box-shadow';
 import {
   addGradientBackgroundListener,
   gradientBackgroundGenerator,
 } from './pages/gradient-background';
-import {addAnimationListener, animationGenerator} from './pages/animation';
 import {
-  addBorderRadiusListener,
-  borderRadiusGenerator,
-} from './pages/border-radius';
-import {boxShadowGenerator, addBoxShadowListener} from './pages/box-shadow';
-import {addTextShadowListener, textShadowGenerator} from './pages/text-shadow';
+  addGradientBorderListener,
+  gradientBorderGenerator,
+} from './pages/gradient-border';
+import {
+  addGradientTextListener,
+  gradientTextGenerator,
+} from './pages/gradient-text';
 import {rangeGenerator} from './pages/input-range';
+import {picTextGenerator} from './pages/pic-text';
```

Comment:
i don't know/understand why you changed the position of the import statements, feels like an unnecessary change

Reply:
I think my prettier did this, I didn't manually move any import statements around